db/migration/0013_previous_lenny_balinger.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "community_district" ALTER COLUMN "li_ft" SET DATA TYPE geometry(multiPolygon,2263);
```

Comment:
```suggestion
UPDATE "community_district" SET "li_ft" = NULL; --> statement-breakpoint
ALTER TABLE "community_district" ALTER COLUMN "li_ft" SET DATA TYPE geometry(multiPolygon,2263);
```

Reply:
Updated. To confirm, when we want to do things like drop columns, we should manually edit the migration files?

Reply:
I find it's a grey area on whether it's better to create [a custom migration file](custom) from scratch or edit an auto-generated one. My guideline is that it's okay to edit the auto-generated migration as long as it doesn't contradict the snapshot meta-data.

It's important to highlight that we never [`DROP`](https://www.w3schools.com/SQL/sql_ref_drop_column.asp) the "li_ft" column. That would push us into "temporarily contradicting" the snapshot meta-data. We only update the content of the rows to remove any errant data that is incompatible with the incoming data structure.

Reply:
I strongly recommend against manually editing generated migrations. That's generally a big no-no for ORMs with migration generators and I don't see why Drizzle would be any different.

Reply:
Yeah, we should  only fix a migration with another migration.

Reply:
Can you expand on why it's a "no-no"?

Reply:
My initial hesitation was simply on the principle of not wanting to mess with automatically generated files. I'm unfamiliar with cross referencing the snapshot metadata. I see that we can use something like the `--custom` flag with Drizzle to generate an empty migration file for manual SQL (maybe same as what Tim linked, can't see it rn). I'm fine either way but would be good to agree on as a team for how we wanna deal

Reply:
@pratishta, I'm not sure whether you saw in chat. But, we were talking about actually dropping columns as we're doing two migrations anyway.

Reply:
Yup

Reply:
Did you have questions or concerns? It appears we're still "updating+altering" instead of "dropping+adding"